Sources/EpoxyCollectionView/CollectionView/ReusableViews/CollectionViewCell.swift

```diff
@@ -132,6 +132,11 @@ public final class CollectionViewCell: UICollectionViewCell, ItemCellView {
   weak var accessibilityDelegate: CollectionViewCellAccessibilityDelegate?
   var ephemeralViewCachedStateProvider: ((Any?) -> Void)?
 
+  /// The item path of the cell from its last configuration update. Used to associate the view with the underlying data. When collection
+  /// view provides view display callbacks, if it is mid update, we need this to see if the view came from pre-update data or
+  /// post-update data.
+  var itemPath: ItemPath?
```

Comment:
We should probably make the same fix in `CollectionViewReusableView` for supplementary views

Reply:
fast follow, see above comment ;)

Reply:
love a fast follow - SGTM!